nodes/docker-compose.yaml

```diff
@@ -11,28 +11,30 @@ services:
       - ../pathfinder-var.env
     restart: unless-stopped
     command: [
-      "--network",
-      "sepolia-testnet",
-      "--p2p.identity-config-file",
-      "identity1.json",
-      "--p2p.listen-on",
-      "/ip4/0.0.0.0/tcp/20002",
-      "--debug.restart-delay",
-      "5",
-      "--debug.pretty-log",
-      "true",
-      # Behave as a proxy into the sequencer gateway, fetching blocks
-      # from the centralized sequencer and storing them.
-      "--p2p.proxy",
-      "true",
-      # The RPC isn't used inside the docker network, so it's disabled by default.
-      # If you wish to enable it, remember to also expose the RPC port.
-      "--rpc.enable",
-      "false"
-    ]
+        "--network",
+        "sepolia-testnet",
+        "--ethereum.url",
+        "wss://sepolia.infura.io/ws/v3/58fd6002745048a3a505ffa9aa0a5c69",
```

Comment:
Uh, this is leaking your supposedly private URL here. I guess this is _typically_ something that should be set in the `pathfinder-var.env` file instead and kept out of the repo?